compiler/ilgen/MethodBuilder.hpp

```diff
@@ -27,8 +27,10 @@
 #endif
 
 
+#include <map>
 #include <fstream>
 #include "ilgen/IlBuilder.hpp"
+#include "env/TypedAllocator.hpp"
```

Comment:
You should really add env/TypedAllocator.hpp and its dependent files to jitbuilder/build/rules/common.mk (the list of copied header files) in this commit. Otherwise, isn't jitbuilder broken with this commit because TypedAllocator.hpp won't be on the usual include path?